ipalib/constants.py

```diff
@@ -340,3 +341,41 @@
 )
 
 SOFTHSM_DNSSEC_TOKEN_LABEL = u'ipaDNSSEC'
+# Apache's mod_ssl SSLVerifyDepth value (Maximum depth of CA
```

Comment:
this shouldn't be here and only # vault data wrapping algo section, will fix it.